static/src/javascripts/projects/common/views/ui/notifications-follow-link.html

```diff
@@ -1,10 +1,10 @@
- <button class="button button--large button--tertiary js-notifications__button notifications__button notifications-follow-input--solo notifications__button--<%= isSubscribed ? 'subscribed' : 'unsubscribed' %>" data-link-name="live-blog-notifications-turned-<%= isSubscribed ? 'off' : 'on' %>">
-     <%= icon %>
+ <button class="js-notifications__toggle notifications__toggle notifications-follow-input--solo notifications__toggle--<%= isSubscribed ? 'subscribed' : 'unsubscribed' %>" data-link-name="live-blog-notifications-turned-<%= isSubscribed ? 'off' : 'on' %>">
      <% if (isSubscribed) { %>
-     <span class="live-notifications__label live-notifications__label--subscribed">Alerts on for this story</span>
+         <div class="live-notifications__slider live-notifications__slider--subscribed"></div>
+         <span class="live-notifications__label live-notifications__label--subscribed">Receive updates on this story</span>
      <% } else { %>
-     <span class="live-notifications__label live-notifications__label--unsubscribed">Get alerts for this story</span>
+         <div class="live-notifications__slider live-notifications__slider--unsubscribed"></div>
+         <span class="live-notifications__label live-notifications__label--unsubscribed">Receive updates on this story</span>
```

Comment:
It's also not possible to tab to these elements to enable keyboard users to turn notifications on and off - maybe we could use a radio button and labels under the hood to ensure it's accessible? Paypal have a good example: https://www.paypal-engineering.com/2014/01/15/a-sweet-toggle-switch/